resources/mediaplayer/src/main/java/org/mobicents/media/server/impl/resource/mediaplayer/audio/wav/WavTrackImpl.java

```diff
@@ -168,80 +154,70 @@ private void getAudioFormat(ReadableByteChannel rbc) throws IOException {
         if (hasFact) {
             // skip fact chunk
             sizeOfData -= 12;
```

Comment:
Where does this magic number 12 comes from? Can we make it a constant and explain why it assumes this particular value?